client/my-sites/site-settings/masterbar.jsx

```diff
@@ -0,0 +1,81 @@
+/**
+ * External dependencies
+ */
+import React, { PropTypes } from 'react';
+import { localize } from 'i18n-calypso';
+import { connect } from 'react-redux';
+
+/**
+ * Internal dependencies
+ */
+import Card from 'components/card';
+import JetpackModuleToggle from 'my-sites/site-settings/jetpack-module-toggle';
+import FormFieldset from 'components/forms/form-fieldset';
+import { getSelectedSiteId } from 'state/ui/selectors';
+import { isJetpackModuleUnavailableInDevelopmentMode, isJetpackSiteInDevelopmentMode } from 'state/selectors';
+import InfoPopover from 'components/info-popover';
+import ExternalLink from 'components/external-link';
+import QueryJetpackConnection from 'components/data/query-jetpack-connection';
+
+const Masterbar = ( {
+	isRequestingSettings,
+	isSavingSettings,
+	selectedSiteId,
+	masterbarModuleUnavailable,
+	translate
+} ) => {
+	return (
+		<div>
+			<QueryJetpackConnection siteId={ selectedSiteId } />
+
+			<Card className="masterbar__card site-settings__security-settings">
+				<FormFieldset>
+					<div className="masterbar__info-link-container site-settings__info-link-container">
+						<InfoPopover position={ 'left' }>
+							<ExternalLink href={ 'https://jetpack.com/support/masterbar/' } icon target="_blank">
+								{ translate( 'Learn more about WordPress.com Toolbar.' ) }
```

Comment:
Hi! I've found a possible matching string that has already been translated **21** times:
`translate( 'Learn more about WordPress.com' )` *ES Score: 8*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).